recipes/objectpath/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "objectpath" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 461263136c79292e42431fbb85cdcaac4c6a256f6b1aa5b3ae9316e4965ad819
+
+build:
+  number: 0
+  entry_points:
+    - objectpath = objectpath.shell:main
+  script: "{{ PYTHON }} -m pip install . -vv "
```

Comment:
Can this be `noarch: python`?

Reply:
No it cannot, because we are using pre-processing selectors in requirements -> run section,

Reply:
Ahh OK